frontend/src/search.tsx

```diff
@@ -0,0 +1,76 @@
+import { TiDeleteOutline } from 'react-icons/ti'
+import { BiRightArrowAlt } from 'react-icons/bi'
```

Comment:
This package is not in the package.json

As a result, this causes a compilation error and the app cannot be run